tests/util.ts

```diff
@@ -12,6 +12,6 @@ export const assertDeepMatchAndLength = (a: any, b: any) => {
 		expect(a).to.deep.equal(b);
 	}
 
-	expect(a).to.deep.match(b);
+	expect(a).to.deep.equals(b);
```

Comment:
`match` is intended for regexps, based on the usage of this `deep.equals` looked like the correct/valid replacement

Reply:
@Page- `deep.match` is actually added by the `chai-samsam` module.
it's essentially a partial checking function and is needed here.